src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -110,5 +111,26 @@ inline arm_compute::DataLayout getAclDataLayoutByMemoryDesc(MemoryDescCPtr desc)
     return arm_compute::DataLayout::UNKNOWN;
 }
 
+/**
+* @brief Return static mutex for lock ComputeLibrary configuration
+* @return static std::mutex
+*/
+static arm_compute::Mutex & getMutexConfig() {
```

Comment:
Abbreviation Config in this case is misleading.
Let's have just getConfigurationMutex

Reply:
@EgorDuplensky corrected it